pkg/reconciler/v1alpha1/taskrun/taskrun_test.go

```diff
@@ -334,6 +334,33 @@ func TestReconcile(t *testing.T) {
 				},
 			},
 		},
+	}, {
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "test-taskrun-with-taskSpec",
+			Namespace: "foo",
+		},
+		Spec: v1alpha1.TaskRunSpec{
+			TaskSpec: &v1alpha1.TaskSpec{
+				Inputs: &v1alpha1.Inputs{
+					Params: []v1alpha1.TaskParam{{
+						Name:        "myarg",
+						Description: "mydesc",
+						Default:     "mydefault",
+					}},
```

Comment:
looks like maybe we DO support params! i cant see a strong reason not support them esp. if we get it for free